executor/infoschema_reader.go

```diff
@@ -1426,6 +1427,125 @@ func (e *memtableRetriever) setDataFromTableConstraints(ctx sessionctx.Context,
 	e.rows = rows
 }
 
+// tableStorageStatsRetriever is used to read slow log data.
+type tableStorageStatsRetriever struct {
+	dummyCloser
+	table         *model.TableInfo
+	outputCols    []*model.ColumnInfo
+	retrieved     bool
+	initialized   bool
+	extractor     *plannercore.TableStorageStatsExtractor
+	initialTables []*initialTable
+	curTable      int
+	pdAddress     string
```

Comment:
pdAddress is useless?

Reply:
`pdAddress` is used to cache the pdAddress to reuse the address in retrieve.

Reply:
Em... But it was never been readed.

Reply:
Ping~ @reafans

Reply:
Sorry i missed it, now removed the useless variable.